content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
+   :alt: Internal transfer form for products ordered from vendor.
 
-Process a delivery order in two steps (pick + ship)
-===================================================
+Process delivery order in two steps (pick + ship)
+=================================================
 
-Create a sales order
---------------------
+When products are delivered in two steps, they move from warehouse stock to an output location, then
+from the output location to a customer location in the database upon validation of a picking order,
+and a subsequent delivery order (DO).
 
-In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select
-(or create) a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
-click :guilabel:`Confirm`.
+Create sales order
+------------------
 
-After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart button will appear in
-the top, above the :abbr:`SO (Sales Order)` form. Click the :guilabel:`Delivery` smart button to
-reveal the associated receipt.
+To create a |SO|, navigate to the :menuselection:`Sales app`, and click :guilabel:`New`. This
+opens a blank sales quotation form.
 
-.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
+Add a customer in the :guilabel:`Customer` field. Then, fill out the various fields on the sales
+quotation form, as necessary.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-new-sales-order.png
    :align: center
-   :alt: Delivery smart button on sales order form.
+   :alt: Filled out new sales order form.
 
-.. tip::
-   Sales order receipts can also be found in the :menuselection:`Inventory` application. In the
-   :guilabel:`Overview` dashboard, click the :guilabel:`# To Process` smart button in the
-   :guilabel:`Pick` Kanban card.
+Under the :guilabel:`Product` tab, click :guilabel:`Add a product`, and select a product to add to
+the sales order quotation.
+
+Once ready, click :guilabel:`Confirm`. This moves the quotation to the :guilabel:`Sales Order`
+stage.
+
+Once the |SO| is confirmed, a :guilabel:`Delivery` smart button appears at the top of the form.
+Clicking the smart button opens the warehouse delivery (WH/OUT) form.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
+   :align: center
+   :alt: Delivery smart button on validated sales order form.
 
-Process the picking
--------------------
+Process picking
+---------------
 
 Once the sales order is confirmed, a picking order is generated and ready to process.
 
-Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
-:guilabel:`Inventory Overview` dashboard. This reveals a list of all pickings to process.
+To complete the picking, navigate to the :guilabel:`Inventory app`, and locate the :guilabel:`Pick`
+task card on the :guilabel:`Inventory Overview` dashboard.
```

Comment:
```suggestion
To complete the picking, navigate to the :guilabel:`Inventory app`, and locate the :guilabel:`Pick`
task card on the :guilabel:`Inventory Overview` dashboard.
```

Alternatively, can I click the smart button?

Reply:
Yes! Added that in for clarification. Suggestion implemented ✅